ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts

```diff
@@ -0,0 +1,33 @@
+export enum BuiltinButtonTypes {
+  processSave = "process-save",
+  deploy = "deploy",
+  deployCanel = "deploy-canel",
+  deployMetrics = "deploy-metrics",
+  editUndo = "edit-undo",
+  editRedo = "edit-redo",
+  editCopy = "edit-copy",
+  editPaste = "edit-paste",
+  editDelete = "edit-delete",
+  editLayout = "edit-layout",
+  groupStart = "group-start",
+  groupFinish = "group-finish",
+  groupCancel = "group-cancel",
+  groupUngroup = "group-ungroup",
+  editProperties = "edit-properties",
+  processCompare = "process-compare",
+  processMigrate = "process-migrate",
+  processImport = "process-import",
+  processJSON = "process-JSON",
+  processPDF = "process-PDF",
```

Comment:
Let's be consistent: process-pdf?

Reply:
this string will be changed in last step